docs/usage/interpreter/interpreter_binding_mode.md

```diff
@@ -25,4 +25,88 @@ limitations under the License.
 
 ## Overview
 
+Interpreter is a JVM process that communicates to Zeppelin daemon using thrift. 
+Each Interpreter process can have Interpreter Groups, and each interpreter instance belongs to this Interpreter Group.
+(See [here](../../development/writing_zeppelin_interpreter.html) to understand more about its internal structure.) 
 
+Zeppelin provides 3 different modes to run interpreter process: **shared**, **scoped** and **isolated**.   
+Also, user can specify the scope of this mode as well: **per user** or **per note**.  
+These 3 modes give flexibility to fit Zeppelin into any type of use cases.
+
+In this documentation, we mainly discuss the combination of **per note** mode with **shared**, **scoped** and **isolated** modes for explanation. 
+
+## Shared Mode
+
+<div class="text-center">
+    <img src="/assets/themes/zeppelin/img/docs-img/interpreter_binding_mode-shared.png">
+</div>
+<br/>
+
+In **Shared** mode, single JVM process and single Interpreter Group serves all Notes.
+
+## Scoped Mode
+
+<div class="text-center">
+    <img src="/assets/themes/zeppelin/img/docs-img/interpreter_binding_mode-scoped.png">
+</div>
+<br/>
+
+In Scoped mode, Zeppelin still runs single interpreter JVM process but multiple Interpreter Group serve each Note. 
```

Comment:
>> Zeppelin still runs single interpreter JVM process but multiple Interpreter Group serve each Note. 

This is only correct when set scoped per note. As in another comment, we should mention in which dimension the scope is. And what does it mean for Per User and Per Note. The same for isolated mode.